### ISSUE TITLE
Should .webkitEntries be pppulated for dialog selections?

### ISSUE THREAD

**Issue by pipcet** [WICG/entries-api#15]
Right now, Firefox populates an input element's `.webkitEntries` only when files are dragged-and-dropped onto the <input> element, not when the file dialog is clicked and a file manually selected: https://bugzilla.mozilla.org/show_bug.cgi?id=1326031

Chrome behavior is similar, but after a file has been dragged-and-dropped, the selection can be changed in the dialog and the change will be reflected in `.webkitEntries`.

Both browsers' behavior appears to me to be in violation of the spec, and to present an accessibility issue as drag-and-drop is often cumbersome to use. Am I misreading this?

**Comment by Robinson-George** [comment]
Yes, definitely.  It is ridiculous that a directory tree can be selected by a Drag&Drop operation but not by the File Picker Dialog.

See what kind of problems this is causing:
https://stackoverflow.com/questions/58384785/obtaining-a-directory-tree-from-the-input-type-file-button

**Comment by inexorabletash** [comment]
> Both browsers' behavior appears to me to be in violation of the spec

Well, this particular spec is trying to document the behavior of browsers, not set a direction. But the spec doesn't make it clear that browsers are weird here...

> and to present an accessibility issue as drag-and-drop is often cumbersome to use. 

Yep - it's not good. Browsers should do better here...

> See what kind of problems this is causing:

:cry:

**Comment by omgoshjosh** [comment]
Dang almost 5 years no resolution??
